app/controllers/users/folders_controller.rb

```diff
@@ -12,7 +12,11 @@ def show
   end
 
   def new
-    @folder = Folder.new
+    if current_folder.global? || current_folder.root_global?
+      @folder = Folder.new(permission: 'global')
```

Comment:
could we pass the `current_folder` in with the new Folder object rather than making it a hidden value in the form?